hack/build-images.sh

```diff
@@ -78,39 +72,78 @@ cp -pf "${OS_ROOT}/contrib/systemd/openshift-sdn-ovs.conf" images/node/conf/
 
 # builds an image and tags it two ways - with latest, and with the release tag
 function image {
-  local STARTTIME=$(date +%s)
-  echo "--- $1 ---"
-  build $1:latest $2
-  #docker build -t $1:latest $2
-  docker tag $1:latest $1:${OS_RELEASE_COMMIT}
-  git clean -fdx $2
-  local ENDTIME=$(date +%s); echo "--- $1 took $(($ENDTIME - $STARTTIME)) seconds ---"
-  echo
-  echo
+  local tag=$1
+  local dir=$2
+  local out
+  out="$( mktemp )"
+
+  set +e
```

Comment:
To capture exit code.  Is there a more succinct sub shell exit code capture that doesn't prevent exit?  Didn't want a multi-line if condition, but if that's cleaner i'm fine with it.